CHANGES.txt

```diff
@@ -7,15 +7,18 @@ V3.0.19
  - A dramatic plugin import error will be tolerated. Plugin will not be loaded.
  - Possible outputs definition is now possible. Protocols can define possible outputs to allow designing
    a workflow in advance without executing them.
+ - Defining the variable SCIPION_STEPS_CHECK_SEC in order to control the time in
```

Comment:
What about htis?
New variable added: SCIPION_STEPS_CHECK_SEC. Defaults to 3 (seconds). It represents how often new input is checked in streamified protocols. Increasing this value will relax the stress in the system, but will feel less "alive".